SDK/articles/CreatingAConnectedServiceExtension.md

```diff
@@ -148,7 +143,7 @@ Stop debugging, and add a new class named **Handler** to the root of your projec
 	namespace ConnectedServiceSample
 	{
 		[ConnectedServiceHandlerExport("Contoso.SampleService",
-		AppliesTo = "CSharp+Web")]
+		AppliesTo = "CSharp | Web")]
```

Comment:
This seems like an odd AppliesTo expression, to work with *any* Web project and also work with any C# project. So you work with console apps, class libraries, etc. and also VB web projects.
I think the *and* operator used before made more sense. Why the change?